datarobot_provider/_experimental/__init__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2025 DataRobot, Inc. and its affiliates.
+#
+# All rights reserved.
+#
+# This is proprietary source code of DataRobot, Inc. and its affiliates.
+#
+# Released under the terms of DataRobot Tool and Utility Agreement.
+import logging
+
+logger = logging.getLogger(__package__)
+
+experimental_warning = (
+    "You have imported from the _experimental directory.\n"
+    "This directory is used for unreleased datarobot features.\n"
+    "Unless you specifically know better,"
+    " you don't have the access to use this functionality in the app, so this code will not work."
+)
```

Comment:
This doesn't read quite right to me, because my assumption is that "access" isn't the issue -- that is, feature flags will not be the reason for things not working. Maybe:

```suggestion
experimental_warning = (
    "You have imported from the _experimental directory.\n"
    "This directory is used for unreleased datarobot features.\n"
    "Code from this directory may be untested and may change without warning."
)
```

Reply:
We could call in the Docs. reinforcements as well 😄 

@nate-goudreault if you feel like chiming in here.

Reply:
```suggestion
experimental_warning = (
    "You have imported from the _experimental directory.\n"
    "This directory is used for unreleased DataRobot features.\n"
    "Unless you specifically know better,"
    " you don't have the access to use this functionality in the app, so this code will not work."
)
```